gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/CseValidNetworkShifter.java

```diff
@@ -0,0 +1,105 @@
+
+    public NetworkShifter getNetworkShifterReduceToFranceAndItaly(TTimestampWrapper timestampWrapper,
+                                                                  Network network,
+                                                                  String glskUrl) {
+        GlskDocument glskDocument = fileImporter.importGlsk(glskUrl);
+        return new LinearScaler(
+                glskDocument.getZonalScalable(network),
+                new SplittingFactors(getExportCornerSplittingFactorsMapReduceToFranceAndItaly(timestampWrapper)),
+                SHIFT_TOLERANCE);
+    }
+
+    NetworkShifter getNetworkShifterWithShiftingFactors(TTimestampWrapper timestampWrapper,
+                                                        Network network,
+                                                        String glskUrl) {
+        GlskDocument glskDocument = fileImporter.importGlsk(glskUrl);
+        return new LinearScaler(
+                glskDocument.getZonalScalable(network),
+                new SplittingFactors(getExportCornerSplittingFactors(timestampWrapper)),
+                SHIFT_TOLERANCE);
+    }
+
+    Map<String, Double> getImportCornerSplittingFactors(TTimestampWrapper timestampWrapper) {
+        Map<String, Double> splittingFactorsMap = timestampWrapper.getImportCornerSplittingFactors();
+        splittingFactorsMap.put(eicCodesConfiguration.getItaly(), -1.);
+        return splittingFactorsMap;
+    }
+
+    Map<String, Double> getExportCornerSplittingFactorsMapReduceToFranceAndItaly(TTimestampWrapper timestampWrapper) {
+        Map<String, Double> result = new HashMap<>();
+        double franceFactor = timestampWrapper.isExportCornerActiveForFrance() ? -1.0 : 1.0;
+        result.put(eicCodesConfiguration.getFrance(), franceFactor);
+        result.put(eicCodesConfiguration.getItaly(), franceFactor * -1);
+        return result;
+    }
+
+    Map<String, Double> getExportCornerSplittingFactors(TTimestampWrapper timestampWrapper) {
+        return timestampWrapper.getExportCornerSplittingFactors().entrySet().stream()
+                .collect(Collectors.toMap(
+                    Map.Entry::getKey,
+                    stringDoubleEntry -> stringDoubleEntry.getValue() * getFactorSignOfCountry(timestampWrapper, stringDoubleEntry.getKey())
+                ));
+    }
+
+    private double getFactorSignOfCountry(TTimestampWrapper timestampWrapper, String countryEic) {
+        return timestampWrapper.isExportCornerActiveForCountry(countryEic) ? -1 : 1;
+    }
+
+    public void shiftNetwork(double shiftValue, Network network, TTimestampWrapper timestampWrapper, String glskUrl) {
+        NetworkShifter networkShifter = getNetworkShifterWithShiftingFactors(timestampWrapper, network, glskUrl);
+        try {
+            networkShifter.shiftNetwork(shiftValue, network);
+        } catch (GlskLimitationException | ShiftingException e) {
+            throw new CseValidShiftFailureException(e.getMessage());
+        }
+    }
```

Comment:
I would rename the class NetworkShifterProvider and move this method to the newly created class ExportCornerComputation so that responsibility of each makes more sense